src/main/java/tracko/logic/commands/order/MarkOrderCommand.java

```diff
@@ -0,0 +1,81 @@
+package tracko.logic.commands.order;
+
+import static java.util.Objects.requireNonNull;
+import static tracko.logic.parser.CliSyntax.FLAG_DELIVERED;
+import static tracko.logic.parser.CliSyntax.FLAG_PAID;
+
+import javafx.collections.ObservableList;
+import tracko.commons.core.Messages;
+import tracko.commons.core.index.Index;
+import tracko.logic.commands.Command;
+import tracko.logic.commands.CommandResult;
+import tracko.logic.commands.exceptions.CommandException;
+import tracko.model.Model;
+import tracko.model.order.Order;
+
+/**
+ * Marks the order identified with it's displayed index in the displayed order list as delivered and/or paid.
+ */
+public class MarkOrderCommand extends Command {
+
+    public static final String COMMAND_WORD = "marko";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Marks the order identified by the index number used in the displayed order list.\n"
+            + "Parameters: INDEX (must be a positive integer) " + FLAG_PAID + " (optional) "
+            + FLAG_DELIVERED + " (optional) \n"
+            + "Example: " + COMMAND_WORD + " 1 " + FLAG_PAID + " " + FLAG_DELIVERED;
+
+    public static final String MESSAGE_MARK_ORDER_SUCCESS = "Marked Order:\n %1$s";
```

Comment:
small nitpick, but adding a space after `\n` will make it look like

Marked Order:
` `Name:
Email
Address:
etc.

Reply:
oh gosh, thanks for pointing that out! I'll make the changes in the upcoming PR :)